examples/with-semantic-ui/pages/index.js

```diff
@@ -0,0 +1,38 @@
+import Head from 'next/head'
+import { Modal, Header, Button, Image, List } from 'semantic-ui-react'
+
+export default () => (
+  <div>
+    <Head>
+      <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.2.2/semantic.min.css" />
```

Comment:
🤔 I think this can be inside `pages/_document.js`, that way it's loaded in every page, if you use an UI framework it's most possible that you want to use it everywhere.